### ISSUE TITLE
Do These Sample Show Video Playing in a Client?

### ISSUE THREAD

**Issue by keithnicholson** [microsoftgraph/microsoft-graph-comms-samples#335]
This is really a question and follow-up to #53.

Is there a sample of using a client against the API's.  I need a real simple sample of doing a 1 to 1 call via a XAML type solution, just to show as proof of concept.

**Comment by InDieTasten** [comment]
No, but the HueBot sample will be the closest sample to what you want to achieve.
It's decoding incoming video, tainting it red, green or blue, and then streaming it back into the meeting as web cam stream, like a colored mirror.

Instead of coloring the frames, you could display them in your desktop application. Running a bot is quite the endeavor just as a heads up. Your client will need to have a domain, certificate and open ports to work.

This use case is also a bit of a misuse of the Bot registration, as it's not really a bot, but rather a custom client. I'm not sure, whether this is allowed with respect to ToC/ToS.
